nginx-docker.yml

```diff
@@ -0,0 +1,42 @@
+version: "3.8"
+
+services:
+  nginx:
+    image: nginx:latest
+    configs:
+      - source: metax-nginx-config
+        target: '/etc/nginx/nginx.conf'
+      - source: fairdata-ssl-certificate
+        target: '/etc/nginx/ssl_certs/fd-test.csc.fi.crt.pem'
+      - source: fairdata-ssl-certificate-key
+        target: '/etc/nginx/ssl_certs/fd-test.csc.fi.key.pem'
+      - source: metax-nginx-elastic-headers-config
+        target: '/etc/nginx/elastic_headers.conf'
+      - source: metax-nginx-shared-headers-config
+        target: '/etc/nginx/shared_headers.conf'
+      - source: metax-nginx-api-response-headers-config
+        target: '/etc/nginx/api_response_headers.conf'
+      - source: metax-nginx-static-file-headers-config
+        target: '/etc/nginx/static_file_headers.conf'
+      - source: metax-nginx-dh-param-config
+        target: '/etc/nginx/ssl_certs/nginx_dhparam.pem'
+    ports:
+      - 4433:443
+
+configs:
+  fairdata-ssl-certificate:
+    external: True
+  fairdata-ssl-certificate-key:
+    external: True
+  metax-nginx-config:
+    external: True
+  metax-nginx-elastic-headers-config:
+    external: True
+  metax-nginx-shared-headers-config:
+    external: True
+  metax-nginx-api-response-headers-config:
+    external: True
+  metax-nginx-static-file-headers-config:
+    external: True
+  metax-nginx-dh-param-config:
+    external: True
```

Comment:
should this still be in the same compose file now that it is working?

Reply:
I would leave it out, as metax can be run without it too